packages/typegpu/src/data/attributes.ts

```diff
@@ -174,14 +131,12 @@ export function attribute<
  * @param alignment The multiple of bytes this data should align itself to.
  * @param data The data-type to align.
  */
-export function align<
-  TAlign extends number,
-  TData extends AnyTgpuData | AnyTgpuLooseData,
->(alignment: TAlign, data: TData): Decorate<TData, Align<TAlign>> {
-  return attribute(data, { type: 'align', alignment }) as Decorate<
-    TData,
-    Align<TAlign>
-  >;
+export function align<TAlign extends number, TData extends AnyData>(
+  alignment: TAlign,
+  data: TData,
+): Decorate<Exotic<TData>, Align<TAlign>> {
+  // biome-ignore lint/suspicious/noExplicitAny: <tired of lying to types>
+  return attribute(data, { type: '@align', value: alignment }) as any;
```

Comment:
The `: Decorate<Exotic<TData>, Align<TAlign>>` should narrow the type again, so that the public API is type-safe.

<img width="804" alt="image" src="https://github.com/user-attachments/assets/97465f0b-5234-415e-9dc7-18102891d43b">

So the `as any` is only to omit the additional `as Decorate<Exotic<TData>, Align<TAlign>`

Reply:
oh, that's right. thought I tested it and it showed as `any`, but I just needed to rerun the build haha